__init__.py

```diff
@@ -260,18 +268,24 @@ def ask_the_wiki(self, sess: Session):
         if results:
             title = results[0].get("title") or \
                     self.session_results[sess.session_id]["query"]
+            self.session_results[sess.session_id]["image"] = results[0].get("img")
             return title, results[0]["summary"]
         return None, None
 
     def display_wiki_entry(self):
         if not can_use_gui(self.bus):
+            LOG.debug(f"GUI not enabled")
             return
         sess = SessionManager.get()
-        image = self.session_results[sess.session_id].get("image") or self.wiki.get_image(query)
+        image = self.session_results[sess.session_id].get("image") or \
+                self.wiki.get_image(self.session_results[sess.session_id]["query"])
         title = self.session_results[sess.session_id].get("title") or "Wikipedia"
         if image:
             self.session_results[sess.session_id]["image"] = image
-            self.gui.show_image(image, title=title, fill=None, override_idle=20, override_animations=True)
+            self.gui.show_image(image, title=title, fill=None,
+                                override_idle=20, override_animations=True)
+        else:
+            LOG.warning(f"No image in {self.session_results[sess.session_id]}")
```

Comment:
maybe use debug or info?

Reply:
mostly had that in to debug the issue that turned out to be bad lookups in `self.session_results`